fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -3195,7 +3195,7 @@ public void testLoanScheduleWithInterestRecalculation_WITH_REST_SAME_AS_REPAYMEN
         String prepayAmount = String.valueOf(prepayDetail.get("amount"));
         validateNumberForEqualWithMsg("verify pre-close amount", "3551.93", prepayAmount);
         todaysDate = Calendar.getInstance(Utils.getTimeZoneOfTenant());
-        String LOAN_REPAYMENT_DATE = dateFormat.format(todaysDate.getTime());
+        final String LOAN_REPAYMENT_DATE = dateFormat.format(todaysDate.getTime());
```

Comment:
I would actually make this lower case, because it's not a constant.

Reply:
agreed, changing.